slurm/retnet.sh

```diff
@@ -14,27 +14,33 @@ export OMP_NUM_THREADS=$SLURM_CPUS_ON_NODE
 
 # LOAD MODULES, INSERT CODE, AND RUN YOUR PROGRAMS HERE
 nvidia-smi
-mamba activate retnet
-python3 ../train_model.py \
+mamba activate new_retnet
+srun python3 ../train_model_lightning.py \
     --activation-dropout 0.1 \
-    --batch-size 128 \
+    --batch-size 8 \
     --checkpoints \
+    --data-dir ../data \
+    --dataset-dir /grphome/grp_retnet/compute/data \
     --dataset-feature text \
     --dataset-name wikitext \
     --dataset-subset wikitext-2-v1 \
     --device cuda \
     --dropout 0.1 \
-    --embed-dim 128 \
-    --epochs 10 \
-    --ffn-dim 1024 \
+    --embed-dim 64 \
+    --epochs 1 \
+    --ffn-dim 512 \
     --fsdp \
+    --heads 8 \
     --layers 6 \
     --lr 0.001 \
     --model retnet \
-    --heads 8 \
     --rand-seed 42 \
     --seq-len 128 \
     --splits 0.7 0.2 0.1 \
-    --val-freq 3 \
+    --tboard-dir /tmp/tboard_logs \
+    --val-freq 1 \
     --value-embed-dim 128 \
-    --vocab-size 30000 \
+    --vocab-size 20000 \
+    --tokenizer-folder /home/datingey/fsl_groups/grp_retnet/compute/tokenizers/wikitext/ \
```

Comment:
Are we sure that datingey will be running the code on the supercomputer? See comment above for jo288 and searlejj.